client/src/competition/competition-details-modal/CompetitionDetailsModal.style.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'vue-styled-components';
+import Button from '../../common/styledComponents/Button';
+
+export const ControlWrapper = styled('div')`
+  position: relative;
+  padding-top: 5px;
+  padding-bottom: 30px;
+  text-align: left;
+`;
+
+export const ControlLabel = styled('label')`
+  display: block;
+`;
+
+export const CancelButton = styled(Button)`
+  margin-right: 10px;
+`;
+
+export const DetailsFieldError = styled('div')`
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  color: #f15151;
```

Comment:
I find that this color is used here [L80](https://github.com/extreme-startup/questionator/blob/master/client/src/App.vue#L80), so maybe makes sense to create a variable

Reply:
If you don't mind, I will create it in the next pull request, I am fixing conflicts for the third time and I want to merge it when it still green :)